docs/swagger/spec.yml

```diff
@@ -6111,7 +6126,24 @@ definitions:
       token_type:
         type: string
         description: 'Type of token returned by using the refresh token'
-
+  UserScopeLimit:
```

Comment:
same usage comment as before -- highlighting separately because the compiler won't grab it